packages/core/src/lib/create-url.test.ts

```diff
@@ -21,7 +21,7 @@ describe('create-url', () => {
     const config = createSocketConfig(cfg);
 
     expect(createSocketUrl(config)).toBe(
-      `wss://api.hume.ai/v0/evi/chat?${config.auth.type}=${config.auth.value}`,
+      `wss://api.hume.ai/v0/evi/chat?${config.auth.type}=${config.auth.value}&enable_chat_metadata=true`,
```

Comment:
Once we launch resumability, we can track removing this query param and enabling chat metadata by default

Reply:
sounds good, once we remove the param let me know, and we can remove the relevant line from the SDK